packages/commonwealth/server/api/external-router.ts

```diff
@@ -141,6 +150,11 @@ const api = {
   getTokenInfoAlchemy,
   getLaunchpadTrades,
   launchTokenBot,
+  createPoll,
+  deletePoll,
+  createPollVote,
+  getPolls,
```

Comment:
queries usually need this pattern unless they are already secure  

getTokens: trpc.query(Token.GetLaunchpadTokens, trpc.Tag.Token, {
    forceSecure: true,
  }),

Reply:
see reviewed